zerver/tests/tests.py

```diff
@@ -230,6 +237,19 @@ def test_admin_restrictions_for_changing_realm_name(self):
         result = self.client_patch('/json/realm', req)
         self.assert_json_error(result, 'Must be a realm administrator')
 
+    def test_admin_restrictions_for_changing_realm_name(self):
+        # type: () -> None
+        new_name = 'Mice will play while the cat is away'
+
+        email = 'othello@zulip.com'
+        self.login(email)
+        user_profile = get_user_profile_by_email(email)
+        do_change_is_admin(user_profile, False)
+
+        req = dict(name=ujson.dumps(new_name))
+        result = self.client_patch('/json/realm', req)
+        self.assert_json_error(result, 'Must be a realm administrator')
```

Comment:
I'm not sure why you copied this test?  I think you don't need it or anything similar to it; see my other comments for notes on where you should add tests.